### DISCUSSION TITLE
Feature request: Custom Array item Actions

### DISCUSSION THREAD

**Discussion by fabien** [sanity-io/sanity#3392] [Sanity Studio Feedback]
I was wondering if the popover menu actions can be customized in `v3`. For example, overriding the `Clear` action here:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/2749/177115045-8429546c-2f28-413c-a52d-9b81ef27aa6b.png">

Or adding other actions, perhaps. I guess this will also be possible once we get to see how to implement a custom input component for arrays (and their items). Unfortunately, there's no documentation yet, and no example code to dissect.

**Comment by TautvydasDerzinskas** [comment]
Any news?

**Comment by Ev1ch** [comment]
Hello. Very useful feature, is there any way to accomplish this?
